mslice/plotting/plot_window/slice_plot.py

```diff
@@ -150,24 +147,26 @@ def update_legend(self):
 
     def change_axis_scale(self, colorbar_range, logarithmic):
         current_axis = self._canvas.figure.gca()
-        images = current_axis.get_images()
-        if len(images) != 1:
-            raise RuntimeError("Expected single image on axes, found " + str(len(images)))
-        mappable = images[0]
+        colormesh = current_axis.collections[0]
         vmin, vmax = colorbar_range
-        if logarithmic and type(mappable.norm) != colors.LogNorm:
-            mappable.colorbar.remove()
-            if vmin == float(0):
+        if logarithmic:
+            label = self.colorbar_label
+            colormesh.colorbar.remove()
+            if vmin <= float(0):
                 vmin = 0.001
```

Comment:
Why `0.001`? Could the colorbar range ever drop below that value?

Reply:
If I remember correctly, ideally this would be zero but that causes a bug in matplotlib so we just use something 'close enough'.